src/main/kotlin/com/emberjs/navigation/EmberGotoClassContributor.kt

```diff
@@ -48,13 +50,27 @@ class EmberGotoClassContributor() : ChooseByNameContributor {
     private fun convert(module: EmberName, file: PsiFile): DelegatingNavigationItem {
         val presentation = DelegatingItemPresentation(file.presentation)
                 .withPresentableText(module.displayName)
-                .withLocationString(null)
+                .withLocationString(getLocation(file))
                 .withIcon(iconProvider.getIcon(module) ?: DEFAULT_ICON)
 
         return DelegatingNavigationItem(file)
                 .withPresentation(presentation)
     }
 
+    private fun getLocation(file: PsiFile): String? {
+        val root = file.virtualFile.parentEmberModule ?: return null
+        val prefix = file.virtualFile.path.removePrefix(root.path)
+
+        return when {
+            root.isInRepoAddon -> "(${root.name} addon)"
+            prefix.startsWith("/app/") -> "(app)"
```

Comment:
I think we should probably keep the app files as the default without any additional prefixes, or maybe only show them when we know that we have different locations?